textproc/libxslt/Portfile

```diff
@@ -96,9 +96,13 @@ if {${name} ne ${subport}} {
                             port:python${python.version} \
                             port:py${python.version}-libxml2
 
+    set python_prefix       ${frameworks_dir}/Python.framework/Versions/${python.branch}
     configure.args-replace  --without-python --with-python
     configure.args-append   --with-python-sys-prefix
     configure.python        ${prefix}/bin/python${python.branch}
+    configure.cflags-append -I${python_prefix}/include/python${python.branch}
+    configure.ldflags-append \
+                            -L${python_prefix}/lib/python${python.branch}/.. -lpython${python.branch}
```

Comment:
These additional flags were needed to get it to build? What error did you get without these?

Can't `-L${python_prefix}/lib/python${python.branch}/..` be simplified to `-L${python_prefix}/lib`?

`-I` flags normally go in CPPFLAGS not CFLAGS. Would that work here?

Reply:
Ultimately these are cribbed from the Python portgroup. Without them it fails at link time (for 3.x only) trying to find all the symbols from the Python interpreter. Libtool tried this:
```
libtool: link: /usr/bin/clang  -o .libs/libxsltmod.so -bundle  .libs/libxsltmod_la-libxslt.o .libs/libxsltmod_la-types.o .libs/libxsltmod_la-libxslt-py.o   -L/opt/local/lib ../libxslt/.libs/libxslt.dylib ../libexslt/.libs/libexslt.dylib -lxml2  -Os -arch x86_64 -Wl,-headerpad_max_install_names -Wl,-syslibroot -Wl,/Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk -arch x86_64 -Wl,-dylib_file -Wl,/opt/local/lib/libxslt.1.dylib:/opt/local/var/macports/build/_Users_david_macports-ports-local_textproc_libxslt/py311-libxslt/work/libxslt-1.1.38/libxslt/.libs/libxslt.dylib
```

And testing it out now, it is working fine with only ldflags defined. Not sure why I was trying out cflags before. I will remove cflags.

Reply:
> I will remove cflags.

And change `-L${python_prefix}/lib/python${python.branch}/..` to `-L${python_prefix}/lib`?